lib/cartopy/crs.py

```diff
@@ -1140,6 +1140,71 @@ def y_limits(self):
         return self._y_limits
 
 
+class LambertEqualArea(Projection):
+    """
+    A Lambert azimuthal equal-area projection.
+
+    """
+
+    def __init__(self, central_longitude=0.0, central_latitude=0.0,
+                 false_easting=0.0, false_northing=0.0,
+                 globe=None):
+        """
+        Kwargs:
+
+            * central_longitude - The central longitude. Defaults to 0.
+            * central_latitude - The central latitude. Defaults to 0.
+            * false_easting - X offset from planar origin in metres.
+                              Defaults to 0.
+            * false_northing - Y offset from planar origin in metres.
+                               Defaults to 0.
+            * globe - A :class:`cartopy.crs.Globe`.
+                      If omitted, a default globe is created.
+
+        """
+        proj4_params = [('proj', 'laea'),
+                        ('lon_0', central_longitude),
+                        ('lat_0', central_latitude),
+                        ('x_0', false_easting),
+                        ('y_0', false_northing)]
+
+        super(LambertEqualArea, self).__init__(proj4_params, globe=globe)
+
+        a = np.float(self.globe.semimajor_axis or 6378137.0)
+        b = np.float(self.globe.semiminor_axis or 6356752.3142)
+        x_axis_offset = 1.3e7 / 6378137.
+        y_axis_offset = 1.3e7 / 6356752.3142
+        self._x_limits = (-a * x_axis_offset + false_easting,
+                            a * x_axis_offset + false_easting)
+        self._y_limits = (-b * y_axis_offset + false_northing,
+                            b * y_axis_offset + false_northing)
```

Comment:
The Y axis limits are a little more complex. It probably wouldn't take too long to figure out the relevant formula, but equally you could just use `transform_points` to get a decent approximation, i.e. if you transform lon=lon_0+180, lat=-lat_0+0.01 the answer will be max_y + false_northing.